### DISCUSSION TITLE
how do I make categories like sections?

### DISCUSSION THREAD

**Discussion by EveryDayRains** [squidfunk/mkdocs-material#2817] [Q&A] [answered]
I need to make "categories" and so that collapsible subsections are available in the categories. Something like gitbook. Is there any way to implement this in mkdocs?
![image](https://user-images.githubusercontent.com/54632865/124639175-84d4d980-de94-11eb-9e18-bec4d415e93f.png)

**Comment by squidfunk** [comment, marked as answer]
I think you might be looking for the following feature flags:

- [`navigation.sections`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-sections)
- [`navigation.indexes`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#section-index-pages)

**Reply by EveryDayRains** [reply]
> I think you might be looking for the following feature flags:
> 
> * [`navigation.sections`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-sections)
> * [`navigation.indexes`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#section-index-pages)


and then how do I make them work as in the screenshot?

**Reply by squidfunk** [reply]
Have you read the linked docs for those features? They explain in detail how to enable them and include screenshots, which should be what you're looking for. Other than that, the styles will be a little different, because what the screenshot you posted shows looks like GitBook, but this could easily be achieved with some extra CSS.

**Reply by EveryDayRains** [reply]
Okay, I'll try, thanks for the answer
